client/src/features/home/NeedsAndOffers.tsx

```diff
@@ -67,8 +67,9 @@ function NeedsAndOffers(props: Props): JSX.Element {
             <Card variant="outlined">
               <img
                 src={
-                  asset?.imgUrls?.[0] ??
-                  'https://optinmonster.com/wp-content/uploads/2019/09/nonprofit-newsletter.png'
+                  asset.imgUrls
+                    ? asset.imgUrls[0]
+                    : 'https://optinmonster.com/wp-content/uploads/2019/09/nonprofit-newsletter.png'
```

Comment:
I think I changed the heading to purple to make sure our theme was still working properly after the upgrade, or its just a result of the theming changes in muiv5.

otherwise this change I don't think I meant to make at all, at least I don't recall why